climada/engine/test/test_forecast.py

```diff
@@ -149,21 +150,41 @@ def test_Forecast_plot(self):
                     for f in source:
                         if f['properties']['adm0_a3'] == 'CHE':
                             sink.write(f)
-        #test plotting functions
+        ## test plotting functions
+        # should save plot without failing
         forecast.plot_imp_map(run_datetime=dt.datetime(2017,12,31),
+                              explain_str='test text',
                               polygon_file=str(cantons_file),
                               save_fig=True, close_fig=True)
         map_file_name = (forecast.summary_str(dt.datetime(2017,12,31)) +
                          '_impact_map' +
                          '.jpeg')
         map_file_name_full = Path(FORECAST_PLOT_DIR) / map_file_name
         map_file_name_full.absolute().unlink(missing_ok=False)
-        forecast.plot_hist(run_datetime=dt.datetime(2017,12,31),
-                           save_fig=False, close_fig=True)
-        forecast.plot_exceedence_prob(run_datetime=dt.datetime(2017,12,31),
-                                      threshold=5000, save_fig=False, close_fig=True)
-
-
+        #should contain title strings
+        ax = forecast.plot_hist(run_datetime=dt.datetime(2017,12,31),
+                           explain_str='test text',
+                           save_fig=False, close_fig=False)
+        title_artists = ax.get_figure().get_children()
+        title_texts = [x.get_text() for x in title_artists if type(x) == plt.Text]
+        self.assertIn('test text', title_texts)
+        self.assertIn('Wed 03 Jan 2018 00-24UTC', title_texts)
+        self.assertIn('31.12.2017 00UTC +3d', title_texts)
+        #should contain average impact in axes
+        artists = ax.get_children()
+        texts = [x.get_text() for x in artists if type(x) == plt.Text]
+        self.assertIn('mean impact:\n 26  USD', texts)
+        ax.get_figure().clf()
+        #should contain title strings
+        ax = forecast.plot_exceedence_prob(run_datetime=dt.datetime(2017,12,31),
+                                      threshold=5000, explain_str='test text exceedence',
+                                      save_fig=False, close_fig=False)[0][0]
+        title_artists = ax.get_figure().get_children()
+        title_texts = [x.get_text() for x in title_artists if type(x) == plt.Text]
```

Comment:
Use `isinstance` for type comparison
```suggestion
        title_texts = [x.get_text() for x in title_artists if isinstance(x, plt.Text)]
```